frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -92,12 +92,8 @@ const hydrateProjects = (state: State, dispatch: React.Dispatch<Action>) => {
       users: string[];
       projects: string[];
     };
-    _.forEach(Object.keys(state[Group.PROJECTS]), p => {
-      // if the project is custom
-      if (state[Group.PROJECTS][p].custom) {
-        requestParams.projects.push(p);
-      }
-    });
```

Comment:
minor nit: maybe do this in another PR since it's unrelated?

Reply:
Adjusted